integration/test/basic/main_test.go

```diff
@@ -63,22 +64,34 @@ func init() {
 		}
 	}
 
+	var k8sClients *k8s.Clients
 	{
-		c := framework.HostConfig{
+		c := k8s.ClientsConfig{
 			Logger: l,
 
-			ClusterID:  "n/a",
-			VaultToken: "n/a",
+			KubeConfigPath: harness.DefaultKubeConfig,
```

Comment:
As follow-up step I'm considering to add something like https://github.com/giantswarm/release-operator/pull/90/files to https://github.com/giantswarm/e2esetup/blob/master/chart/env/common.go to decouple a bit more from harness setup minikube to be able to use kind/e2ectl cluster kubeconfig passed as E2E_KUBECONFIG env var.

Reply:
@sslavic Thanks that is a great next step. Yes we just want to pass in the `E2E_KUBECONFIG` and continue removing the usage of e2e-harness.